tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/test-suite-a/testenum-enum.html

```diff
@@ -1,171 +1,91 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h2>
-        TestEnum
-      </h2>
+      <h2>TestEnum</h2>
       <section>
-        <p>
-          Test Enum
-        </p>
+        <p>Test Enum</p>
       </section>
       <section>
-        <h3 id="testenum-signature">
-          Signature
-        </h3>
-        <code>
-          export declare enum TestEnum
-        </code>
+        <h3 id="testenum-signature">Signature</h3><code>export declare enum TestEnum</code>
       </section>
       <section>
-        <h3 id="testenum-remarks">
-          Remarks
-        </h3>
-        <p>
-          Here are some remarks about the enum
-        </p>
+        <h3 id="testenum-remarks">Remarks</h3>
+        <p>Here are some remarks about the enum</p>
       </section>
       <section>
-        <h3 id="testenum-examples">
-          Examples
-        </h3>
+        <h3 id="testenum-examples">Examples</h3>
         <section>
-          <h4 id="testenum-example1">
-            Example 1
-          </h4>
+          <h4 id="testenum-example1">Example 1</h4>
           <p>
-            <p>
-              Some example
-            </p>
-            <code>
-              const foo = TestEnum.TestEnumValue1
-            </code>
+            <p>Some example</p><code>const foo = TestEnum.TestEnumValue1</code>
           </p>
         </section>
         <section>
-          <h4 id="testenum-example2">
-            Example 2
-          </h4>
+          <h4 id="testenum-example2">Example 2</h4>
           <p>
-            <p>
-              Another example
-            </p>
-            <code>
-              const bar = TestEnum.TestEnumValue2
-            </code>
+            <p>Another example</p><code>const bar = TestEnum.TestEnumValue2</code>
```

Comment:
Not sure I love single-lines for the `<code>` elements. I'm also not sure if those respect formatting like `<pre>` would, so what would happen with multi-line contents (for both). The files I've looked at so far don't have an example of that, might be good to add.

Reply:
The code blocks would include `<br>` tags for line breaks, currently. Good call on adding a multi-line code sample to the test assets, though. I'll put up a separate PR that does that.